driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideOperationsTimeoutProseTest.java

```diff
@@ -74,11 +82,125 @@
  */
 public abstract class AbstractClientSideOperationsTimeoutProseTest {
 
+    private static final String GRID_FS_BUCKET_NAME = "db.fs";
+    private static final String GRID_FS_COLLECTION_NAME_CHUNKS = GRID_FS_BUCKET_NAME + ".chunks";
+    private static final String GRID_FS_COLLECTION_NAME_FILE = GRID_FS_BUCKET_NAME + ".files";
     private final MongoNamespace namespace = new MongoNamespace(getDefaultDatabaseName(), this.getClass().getSimpleName());
     private TestCommandListener commandListener;
     private CollectionHelper<BsonDocument> collectionHelper;
-
+    private CollectionHelper<BsonDocument> filesCollectionHelper;
+    private CollectionHelper<BsonDocument> chunksCollectionHelper;
     protected abstract MongoClient createMongoClient(MongoClientSettings mongoClientSettings);
+    protected abstract GridFSBucket createGridFsBucket(MongoDatabase mongoDatabase, String bucketName);
+
+    @Tag("setsFailPoint")
+    @FlakyTest(maxAttempts = 3)
```

Comment:
Are these tests flaky?

Reply:
The flakiness was due to low timeouts. I've now increased them for better consistency and removed `FlakyTest `annotation.